### ISSUE TITLE
[Question] About the Civilized Jewelers Office 

### ISSUE THREAD

**Issue by GrantSP** [LoneGazebo/Community-Patch-DLL#2234]
The gold maintenance for this office is only 5 compared to all the rest at 6.
Is this by design?

**Comment by LoneGazebo** [comment]
Yep.

**Comment by GrantSP** [comment]
Thanks. Did a bit of fiddling with the wiki and thought I'd ask.

**Comment by pandasnail** [comment]
Hey @GrantSP (aka ExpiredReign, yeah?), on the wiki in general, well, first of all thanks for going through and adding the pretty icons and stuff, I really hate doing that part, I really only like doing the simple entering of data.

Also, tho, if you have the time and energy to do individual pretty pages like the one you did for the Archer, one area that needs it is the Technologies section. A user named CursedSquirrel got maybe halfway (really two fourths in a sense, the beginning and the end of the tree) thru awhile back. I mean, its just my personal opinion of course, but I feel like finishing that out would be a better use of your time than making pages for each military unit. I can be awkward, so if this came out awkward, I'll just preemptively apologize lol.

**Comment by GrantSP** [comment]
I'm actually trying to get it to eventually look like the [CivV wiki](http://civilization.wikia.com/wiki/Civilization_V).
The templates are a bit of a trick to use though, you can modify the template to show whatever you want which is what the archer page is at the moment, just me testing out some stuff.
Sadly the leader page turned out nothing like what I was expecting.
I'm also waiting on the devs to contact me on how to link to all those existing Civ artwork instead of uploading all new ones.

**Comment by pandasnail** [comment]
Not sure what you were trying to do with it, but the Civilizations page is easier to navigate now then before for sure.

Looking at the regular Civ V Wiki, I personally don't like their homepage. Having a very simple navigation home page seems better to me. Now if you mean eventually having way more individual pages with nice templates, I agree that would be better.

**Comment by GrantSP** [comment]
> Now if you mean eventually having way more individual pages with nice templates, I agree that would be better.

This.

**Comment by pandasnail** [comment]
I would kinda say it might be nice to have both tho, like looking at the regular wiki, there's no way to look at atleast some features of a lot of units at once. I think we should still keep the basic CS and cost and so on info on the basic pages, but then it would be neat to have more complicated ideas on a page dedicated to the unit or building or whatever.

**Comment by pandasnail** [comment]
Someone's been busy! Delightful! Still don't see what you see in their Units page tho, think it would be much better to link any new individual unit pages you make thru the page we already have. I know it doesn't have Uniques, but they could be added, either to the Military Units page or their own page. But the level of detail makes it much better then just a list like theirs.

**Comment by GrantSP** [comment]
The reason I'm going for their unit page as they have it is because it uses a template that just adds the correct detail according to what we decide.

Say we change a unit's tech, all we need do is edit it with the template open and the links to that tech, the images and prerequisites are all updated accordingly.
A lot of work now but once it's in there the upkeep is minimal.
I'm also designing some new templates and will add documentation as to how they work later.

At any rate I'm learning as I go and if the majority prefer something else I've left all the existing pages so a return to them is easy.

**Comment by pandasnail** [comment]
Their individual Unit pages are good certainly, I mean the Unit Overview page, theirs lacks any of that sort of detail.

**Comment by GrantSP** [comment]
What detail would you like to see on the unit list page?

**Comment by pandasnail** [comment]
The existing pages for Military Units and Civilian Units.

**Comment by GrantSP** [comment]
No worries. I can edit the template on that page to show a brief summary of each unit. Then you follow the links to get more detail.

**Comment by pandasnail** [comment]
Thanks!

(btw I never made an account on wikia, when you see 'a wikia contributor' its prob me, if you have any thoughts about any edits I make; I basically only update data tho)

**Comment by GrantSP** [comment]
No problems. I was going to get around to updating the values later after all the format is to my liking. I figured at the moment not too many users are using this tool so I could let some errors slip by.
Do what ever you wish, that's the nature of wikis
